src/components/textarea/textarea.pw.tsx

```diff
@@ -0,0 +1,1070 @@
+  });
+
+  (["top", "bottom", "left", "right"] as const).forEach((position) => {
+    test(`should render component with tooltip positioned to the ${position}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <Box m="250px">
+          <TextareaComponent
+            error={CHARACTERS.STANDARD}
+            tooltipPosition={position}
+          />
+        </Box>
+      );
+
+      const getDataElementByValueElementError = getDataElementByValue(
+        page,
+        "error"
+      );
+      await getDataElementByValueElementError.hover();
+      const tooltipPreviewElement = tooltipPreview(page);
+      await waitForAnimationEnd(tooltipPreviewElement);
+
+      await expect(tooltipPreviewElement).toHaveText(CHARACTERS.STANDARD);
+      await expect(tooltipPreviewElement).toHaveAttribute(
+        "data-placement",
+        position
+      );
+    });
+  });
+
+  test("should render with helpAriaLabel prop", async ({ mount, page }) => {
+    await mount(
+      <TextareaComponent
+        labelHelp="field help"
+        helpAriaLabel={CHARACTERS.STANDARD}
+      />
+    );
+
+    await expect(getComponent(page, "help")).toHaveAttribute(
+      "aria-label",
+      CHARACTERS.STANDARD
+    );
+  });
+
+  (["left", "right"] as const).forEach((align) => {
+    test(`should render with align prop set to ${align}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<TextareaComponent align={align} />);
+
+      await expect(textareaChildren(page)).toHaveCSS("text-align", align);
+    });
+  });
+
+  testData.forEach((input) => {
+    test(`should input into and verify the value as ${input}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<TextareaComponent />);
+
+      const textareaChildrenElement = textareaChildren(page);
+      await textareaChildrenElement.fill(input);
+
+      await expect(textareaChildrenElement).toHaveText(input);
+    });
+  });
+
+  testData.forEach((value) => {
+    test(`should render with value prop set to ${value}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<TextareaComponent value={value} />);
+
+      await expect(textareaChildren(page)).toHaveText(value);
+    });
+  });
+
+  ([5, 25, 100] as const).forEach((cols) => {
+    test(`should render with cols prop set to ${cols}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<TextareaComponent cols={cols} />);
+
+      await expect(textareaChildren(page)).toHaveAttribute(
+        "cols",
+        String(cols)
+      );
+    });
+  });
+
+  ([5, 25, 100] as const).forEach((rows) => {
+    test(`should render with rows prop set to ${rows}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<TextareaComponent rows={rows} />);
+
+      await expect(textareaChildren(page)).toHaveAttribute(
+        "rows",
+        String(rows)
+      );
+    });
+  });
+
+  ([
+    [true, 92],
+    [false, 64],
+  ] as const).forEach(([boolean, height]) => {
```

Comment:
```suggestion
  ] as const).forEach(([expandable, height]) => {
```

**suggestion** (non-blocking): as in the other PW refactor I recently reviewed, I prefer to use a descriptive name for booleans like this, rather than just "boolean" or equivalent